packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.9.1
 
+* Updates platform_interface dependency to 2.9.2.
```

Comment:
Why? There are no changes to this package's code.

Reply:
That also looks like some unexpected merge problem. I'm not really sure how or when that change got there.